### PR TITLE
Bump version and CHANGELOG for v3.47.0

### PR DESCRIPTION
**Pull request by DrJosh9000**

### CODE REVIEW

**Review by triarius — DISMISSED** [review]
~~LGTM~~ Sorry, see below

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,36 @@ All notable changes to this project will be documented in this file.
...
+   Unix socket (like the `job-api`).
+
+### Added
+- Experiment: Polyglot hooks [#2040](https://github.com/buildkite/agent/pull/2040) (@moskyb)
+- Experiment: Local Agent API, with locking service [#2042](https://github.com/buildkite/agent/pull/2042) (@DrJosh9000)
+- New flag `--upload-skip-symlinks` (on `artifact upload`) allows skipping symlinks when uploading files. `--follow-symlinks` has been deprecated and renamed to `--glob-resolve-follow-symlinks` [#2072](https://github.com/buildkite/agent/pull/2072) (@triarius)
+
+### Fixed
+- The `normalised-upload-paths` experiment was unintentionally left out of the available experiments list [#2076](https://github.com/buildkite/agent/pull/2076) (@MatthewDolan)
+- The `--no-patch` flag passed to git is replaced with `-s`, to continue supporting older versions of git (pre v1.8.4; 2013) [#2075](https://github.com/buildkite/agent/pull/2075) (@pda)
```
Comment (triarius) [inline CHANGELOG.md:31]: Hmm, this is in the changelog for v3.46.1 already. 
```suggestion
```
Maybe there's some technicality about the git history causing this to appear here, but I think as we released v3.46.1 before this version, we should treat v3.46.1 as an ancestor of this version even if the git commits are not.

Comment (DrJosh9000) [reply CHANGELOG.md]: Yep, I just missed it. Done.

**Review by triarius — APPROVED** [review]
Thanks!

**Pull request merged**